aptos-move/block-executor/src/task.rs

```diff
@@ -163,6 +163,9 @@ pub trait TransactionOutput: Send + Sync + Debug {
     /// Execution output for transactions that comes after SkipRest signal.
     fn skip_output() -> Self;
 
+    /// Execution output for transactions that should be discarded.
+    fn discard_output(discard_code: StatusCode) -> Self;
```

Comment:
Is discarding different from skipping rest of transactions in the block? A comment on when we discard/skip transactions would be helpful.